editor/gui/scene_tree_editor.cpp

```diff
@@ -1602,6 +1629,16 @@ SceneTreeEditor::SceneTreeEditor(bool p_label, bool p_can_rename, bool p_can_ope
 	update_node_tooltip_delay->set_one_shot(true);
 	add_child(update_node_tooltip_delay);
 
+	revoke_dialog = memnew(ConfirmationDialog);
+	add_child(revoke_dialog);
+	revoke_dialog->connect(SceneStringName(confirmed), callable_mp(this, &SceneTreeEditor::_update_ask_before_revoking_unique_name));
+	VBoxContainer *vb = memnew(VBoxContainer);
+	revoke_dialog->add_child(vb);
+	revoke_dialog_label = memnew(Label);
+	vb->add_child(revoke_dialog_label);
+	ask_before_revoke_checkbox = memnew(CheckBox(TTR("Do not ask again")));
```

Comment:
[Use contraction](https://docs.godotengine.org/en/stable/contributing/development/editor/editor_style_guide.html) as it sounds more natural for UI text:

```suggestion
	ask_before_revoke_checkbox = memnew(CheckBox(TTR("Don't ask again")));
```